client/types/logger.go

```diff
@@ -4,4 +4,5 @@ package types
 type Logger interface {
 	Debugf(format string, v ...interface{})
 	Errorf(format string, v ...interface{})
+	Fatalf(format string, v ...interface{})
```

Comment:
I would not want to change the Logger interface which is part of public API. This is a breaking change and also increases the public API size.

Instead I suggest where necessary the example code calls the regular Logger.Errorf() followed by a whatever its preferred termination function is (e.g. os.Exit).

Reply:
ok